jOOQ-scala_2.13/src/main/scala/org/jooq/scalaextensions/Conversions.scala

```diff
@@ -822,4 +821,30 @@ object Conversions {
 
 // [jooq-tools] START [handler]
 // [jooq-tools] END [handler]
+
+  /** Magic to get cross-compiling access to `scala.jdk.CollectionConverters`
+   *  with a fallback on `scala.collection.JavaConverters`, without deprecation
+   *  warning in any Scala version. Credit to
+   *  https://github.com/scala/scala-collection-compat/issues/208#issuecomment-497735669
+   */
+  private object JDKCollectionConvertersCompat {
+    object Scope1 {
+      object jdk {
+        type CollectionConverters = Int
+      }
+    }
+    import Scope1._
+
+    object Scope2 {
+      import scala.collection.{JavaConverters => CollectionConverters}
+      object Inner {
+        import scala._
+        import jdk.CollectionConverters
+        val Converters = CollectionConverters
+      }
+    }
+
+    val Converters = Scope2.Inner.Converters
+  }
+
```

Comment:
That looks laborious. Without an explanation, I don't think that's really necessary.